packages/data/addon/mirage/factories/time-dimension.js

```diff
@@ -16,6 +16,10 @@ export default Factory.extend({
     return `Time Dimension ${this.index}`;
   },
 
+  friendlyName() {
+    return `Friendly Time Dimension ${this.index}`;
```

Comment:
Can we make this the same as the current `name()` value and change `name()` to be the `id()` value without the `table.` prefix?